### PR TITLE
AR21 tweak

### PR DESCRIPTION
**Pull request by aarrisu**
## About The Pull Request
A minor change to the AR21 while making it more viable.

## Why It's Good For The Game
This PR of mine enables the AR21 be able compete with other guns within TGMC. It increases its magazine capacity from 30 to 40, while also boosting its bullet damage from 30 to 32.5. by giving it its own ammo type and also gave it its own box of ammo!!!

## Changelog
:cl:
balance: slightly tweaked ar21's damage and mag size 
/:cl:

### CODE REVIEW

**Comment by Mrrpip** [comment]
Note: By proxy, this also buffs the MG27, which shares the same bullet type.
Perhaps a gun damage modifier might be more practical and less debate worthy?

**Comment by Lumipharon** [comment]
yeah don't just change ammo stats my guy

**Comment by aarrisu** [comment]
woopz

File: code/modules/projectiles/magazines/misc.dm [hunk 1]
```diff
@@ -67,6 +67,15 @@
 	current_rounds = 125
 	max_rounds = 125
 
+/obj/item/ammo_magazine/packet/p10x25smm
```
Comment (Lumipharon) [inline code/modules/projectiles/magazines/misc.dm:70]: wuh, why'd you make a new ammo box?

**Pull request closed without merging**